### ISSUE TITLE
Allow specifying only specific entities from a device in scenes editor.

### ISSUE THREAD

**Issue by ruifung** [home-assistant/frontend#6468]
## The request
In the scene editor, allow only adding specific entities from a device. (i.e Only a specific light/switch entity from a zigbee switch that has multiple relays)

At the time of writing, using the add entity field instead of add device results in the entire device, and all its associated entities being added to the scene, which may not be desired. (i.e. If one of the relays is controlling power to a fan, for example.)

## The alternatives
Editing the scenes.yaml directly to remove the unwanted entities works, but it would be nice to have this be doable from the UI.

**Comment by jtaseff-gfms** [comment]
Upvote for this one. Not intuitive at all

I also don't think entities in scenes should be hidden behind "advanced mode"..... half of my lighting only show up as entities. Such as dual switches, caseta dimmers...

**Comment by kongo09** [comment]
> it appears the generated YAML is indeed correct, and does not include the extra entities

This is not true. The `scenes.yaml` contains all entities, even though only one is selected in the UI using `Add Entity`. At least that's consistent with the behaviour that adding an entity actually adds a device.

**Comment by mangaskahn** [comment]
Please address this issue. My specific use case is Zooz Zen30 double switches. If I add one of the switches as an entity, the whole device gets added. In my garage I want to automate the indoor lights without changing the state of the outdoor lights. Same in my office. I'd like to automate the desk lights without changing the state of the overhead lights. The current GUI process requires me to set a state for both switches on the device.

**Comment by jtaseff-gfms** [comment]
I ran into it recently with another Zooz double outlet too. Somehow it added the "both" entity, so I would click once and they would both come on, click again and they go to the desired state....
@mangaskahn my current workaround is to add the whole device in the UI, then go in with text editor to scenes.yaml and delete the entities I don't need.

**Comment by mangaskahn** [comment]
Thanks, I saw that above. I'll try it.

**Comment by macf0x** [comment]
How has this not been fixed already. The concept of not being able to add a single entity to a scene in the UI is very strange decision and even as it stands the UI is not intuitive at all.

Given one can edit the YAML to get the desire result, there is no technical reason for this.

Can anyone explain why? The last PR has no detail.
